README.md

```diff
@@ -4,56 +4,125 @@
 
 Currently, the researcher or the data user has to answer a number of queries to get the desired results and it often requires iteration. 
 
-The aim of query-tol-ai would be to make this search process more user-friendly by adding an LLM style chatbot interface. This is to be done by utilizing large language models that will be able to interpret the user prompts and initiate the API calls accurately giving the user the desired results.
```

Comment:
Nice catch, I think it's time for us to add codespell to the repo to catch these. See codespell workflow config [here](https://github.com/neurobagel/query-tool/blob/main/.github/workflows/codespell.yml).

Reply:
Yes.. I have created a new PR adding codespell to the workflow #34